LDAP-Auth/LDAP-Auth.csproj

```diff
@@ -13,7 +13,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Jellyfin.Controller" Version="10.4-*" />
+    <PackageReference Include="Jellyfin.Controller" Version="10.5-*" />
```

Comment:
```suggestion
    <PackageReference Include="Jellyfin.Controller" Version="10.*" />
```

Reply:
I'll make a repository we can use to test all plugins before a release so we don't have to deal with these version issues anymore.